pallets/ajuna-tournament/src/impls/fee_handler.rs

```diff
@@ -0,0 +1,37 @@
+use super::*;
+use ajuna_primitives::fee_handler::FeeProvider;
+use sp_arithmetic::traits::{CheckedDiv, Saturating};
+
+impl<T: Config<I>, I: 'static> FeeProvider for Pallet<T, I> {
+	type AccountId = AccountIdFor<T>;
+	type FeeIdentifier = TournamentCategoryIdFor<T, I>;
+	type FeeCurrency = BalanceOf<T, I>;
+	type FeeOutput = (BalanceOf<T, I>, Self::AccountId);
+
+	fn get_fee_from(
+		base_fee: Self::FeeCurrency,
```

Comment:
Same comment as in the other implementation. Do we maybe just want to return the fractions here?

Reply:
Just as in the other comment, I think its better to actually make the calculation at the implementer side.